sdl_android_lib/src/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -1658,18 +1655,21 @@ public void run() {
             	//Log.v(TAG, "Self service info " + self);
             	//Log.v(TAG, "Newest compare to service info " + newestServiceReceived);
             	if(newestServiceReceived!=null && self.isNewer(newestServiceReceived)){
+            		if(SdlRouterService.mSerialService.isConnected()){ //We are currently connected. Wait for next connection 
```

Comment:
It looks like there is a case when this call can cause a null pointer exception.  Please see stacktrace below:

--------- beginning of crash
03-22 14:09:39.997  9740  9740 E AndroidRuntime: FATAL EXCEPTION: main
03-22 14:09:39.997  9740  9740 E AndroidRuntime: Process: com.smartdevicelink.router, PID: 9740
03-22 14:09:39.997  9740  9740 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean com.smartdevicelink.transport.MultiplexBluetoothTransport.isConnected()' on a null object reference
03-22 14:09:39.997  9740  9740 E AndroidRuntime: 	at com.smartdevicelink.transport.SdlRouterService$5.run(SdlRouterService.java:1673)
03-22 14:09:39.997  9740  9740 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:751)
03-22 14:09:39.997  9740  9740 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:95)
03-22 14:09:39.997  9740  9740 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:154)
03-22 14:09:39.997  9740  9740 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6126)
03-22 14:09:39.997  9740  9740 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
03-22 14:09:39.997  9740  9740 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
03-22 14:09:39.997  9740  9740 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
03-22 14:09:40.452   788  1495 I ActivityManager: Killing 9740:com.smartdevicelink.router/u0a272 (adj 800): crash